app/components/avo/fields/boolean_field/edit_component.html.erb

```diff
@@ -1,13 +1,47 @@
 <%= field_wrapper(**field_wrapper_args) do %>
   <div class="h-8 flex items-center">
-    <%= @form.check_box @field.id,
-      value: @field.value,
-      checked: @field.value,
-      class: "text-lg h-4 w-4 checked:bg-primary-400 focus:checked:!bg-primary-400 rounded #{@field.get_html(:classes, view: view, element: :input)}",
-      data: @field.get_html(:data, view: view, element: :input),
-      disabled: disabled?,
-      autofocus: @autofocus,
-      style: @field.get_html(:style, view: view, element: :input)
+    <% 
+      # Common checkbox attributes
+      common_checkbox_attrs = {
+        value: @field.value,
+        checked: @field.value,
+        disabled: disabled?,
+        autofocus: @autofocus,
+        id: "#{@field.id}_checkbox"
+      }
```

Comment:
This can be a method on `app/components/avo/fields/boolean_field/edit_component.rb`